### ISSUE TITLE
Rename the library to [lib]combine

### ISSUE THREAD

**Issue by agarny** [sbmlteam/libCombine#47]
With `[lib]Combine`, a CMake file called `[lib]Combine-static-config.cmake` gets generated. Such a CMake file cannot be found on a case-sensitive system (since that kind of filename should be all lower case). Instead, a CMake file called `[lib]combine-static-config.cmake` should be generated, hence the suggestion to rename the library to `[lib]combine`, not least so that it's consistent with libSBML, for instance.

**Comment by fbergmann** [comment]
The problem that i see, is with this change, all packages that ever used libCombine will have to change. So far they would have had a fallback statement using the `CONFIGS` flag on case sensitive filesystems. But with the PR the number of variables to test would have to be doubled (checking for [lib][c|C]ombine), to test. Here is how other projects were importing in the past: 


```
find_package(${COMBINE_LIBRARY_NAME} CONFIG QUIET)

if (NOT ${COMBINE_LIBRARY_NAME}_FOUND)
  find_package(${COMBINE_LIBRARY_NAME} CONFIG QUIET
  CONFIGS ${COMBINE_LIBRARY_NAME}-config.cmake
  PATHS ...
  )
endif()

```

**Comment by agarny** [comment]
Fair enough. I have my own fork of the library, so I am fine. (Feel free to close this issue and the corresponding PR.)
